public/app/core/internationalization/index.tsx

```diff
@@ -131,5 +130,12 @@ export function getI18next() {
 // Perhaps in the future this will use useTranslation from react-i18next or something else
 // from context
 export function useTranslateInternal() {
-  return t;
+  const context = usePluginContext();
+  if (!context) {
+    return t;
+  }
+
+  const { meta } = context;
+  const pluginT = useMemo(() => i18nInstance.getFixedT(null, meta.id), [meta.id]);
+  return pluginT;
```

Comment:
It could probably make sense to add a test for the scenario when there's a plugin context and check if the namespace is set correctly?

Reply:
Fixed 👍